doc/classes/WorkerThreadPool.xml

```diff
@@ -100,10 +100,13 @@
 			</description>
 		</method>
 		<method name="wait_for_task_completion">
-			<return type="void" />
+			<return type="int" enum="Error" />
```

Comment:
Technically this breaks compatibility. Needs to be assessed whether it's worth making an exception and allow it.

Reply:
I can change it so the exposed API doesn't change. However, I don't really see the downside. If the caller is not interested, they can just ignore the returned value. After all, there's no behavior change. This gives an opportunity to the user of doing something about the deadlock, but the possibility of it occuring was already there.

Reply:
I think its fine, It does not change anything for GDScript and for GDExtension we already have the compatibility fallback system.